libcfnet/client_code.c

```diff
@@ -767,12 +767,6 @@ bool CopyRegularFileNet(const char *source, const char *dest, off_t size,
     NDEBUG_UNUSED int rc = snprintf(cfchangedstr, 255, "%s%s", CF_CHANGEDSTR1, CF_CHANGEDSTR2);
     assert(rc >= 0 && rc < sizeof(cfchangedstr));
 
-    if ((strlen(dest) > CF_BUFSIZE - 20))
-    {
-        Log(LOG_LEVEL_ERR, "Filename too long");
-        return false;
-    }
```

Comment:
It has to do with the CFEngine's (file transfer) protocol, right? I believe `GET $SIZE $FILENAME` have to fit into `CF_BUFSIZE`.

Reply:
But this is the destination file. As far as I can see, it is never sent over the protocol. I don't understand why it is there. It's been there from the beginning of time.

Reply:
Oh, you're right! Then I'm not sure, maybe it was part of some reporting data or whatever. Good riddance now, for sure!